Sources/URLSessionDecodable/URLSessionDecodable.swift

```diff
@@ -28,29 +28,39 @@ extension URLSession {
         completionHandler: @escaping (Result<T, URLSessionDecodableError>) -> Void
     ) -> URLSessionDataTask {
         let request = self.request(with: url, method: method, parameters: parameters, headers: headers)
+        return decodable(request: request, decoder: decoder, completionHandler: completionHandler)
+    }
+    // swiftlint:enable function_parameter_count
+
+    // swiftlint:disable force_unwrapping
+    public func decodable<T: Decodable>(
```

Comment:
1. Missing docs
2. argument labels should be improved: https://www.swift.org/documentation/api-design-guidelines/